manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftBuildTools2015

```

Comment:
```suggestion
PackageIdentifier: Microsoft.BuildTools2015
```

Reply:
Along with this suggestion, the file paths and names will need to be updated to match.